src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts

```diff
@@ -113,6 +181,31 @@ export class DotPageStateService {
             return this.dotContentletLockerService.unlock(workingInode).pipe(pluck('message'));
         }
 
-        return observableOf(null);
+        return of(null);
+    }
+
+    private handleSetPageStateFailed(err: ResponseView): void {
+        this.dotHttpErrorManagerService
+            .handle(err)
+            .pipe(take(1))
+            .subscribe((res: DotHttpErrorHandled) => {
+                if (res.forbidden) {
+                    this.dotRouterService.goToSiteBrowser();
+                } else {
+                    this.reload();
+                }
+            });
+    }
+
+    private setState(state: DotRenderedPageState): void {
+        this.state$.next(state);
+    }
+
+    private shouldLockPageToSetPersona(persona: DotPersona): boolean {
```

Comment:
I think this method name is wrong, should it be something like 'shouldUnlockPageToSetPersona'?, because here we are unlock the page

https://github.com/dotCMS/core-web/pull/1106/files#diff-a75ba4eb5781424fd5bc8909493f7e0cR161